CMakeLists.txt

```diff
@@ -168,6 +168,11 @@ if(ENABLE_PYTHON)
     target_compile_definitions(k40gen PRIVATE "-DXTENSOR_ENABLE_XSIMD")
   endif()
 
+  # xtensor headers
+  find_package(xtensor REQUIRED)
+  target_include_directories(k40gen PUBLIC ${xtensor_INCLUDE_DIRS})
+  target_link_libraries(k40gen PUBLIC xtensor)
```

Comment:
I think xtensor has no libraries, so is this one needed?

Reply:
indeed, it's not needed here